os/board/rtl8730e/src/Makefile

```diff
@@ -213,16 +218,31 @@ $(COBJS) $(LINKOBJS): %$(OBJEXT): %.c
 
 libboard$(LIBEXT): $(OBJS)
 	$(call ARCHIVE, $@, $(OBJS))
+ifeq ($(CONFIG_RTK_DATA_BINARY_TO_EXT_FLASH),y)
+	$(call build_ext_bin)
+else
+	@echo "========== Current config does not support external flash! =========="
+endif
 
 .depend: Makefile $(SRCS)
 	$(Q) $(MKDEP) $(DEPPATH) $(CC) -- $(CFLAGS) -- $(SRCS) >Make.dep
 	$(Q) touch $@
 
 depend: .depend
 
+define build_ext_bin
+	$(Q) $(shell python $(BUILD_DIR)/rtk_data_binary.py)
+	@if [ -d $(RESOURCE_TARGET_FOLDER) ]; then echo "$(RESOURCE_TARGET_FOLDER) exist"; else $(MKDIR) -p $(RESOURCE_TARGET_FOLDER); fi
+	$(Q) mv -f $(BUILD_DIR)/rtk_ext_flash_data.bin $(RESOURCE_TARGET_FOLDER)/
+	@echo "========== Current config support external flash, binary created and moved to path /resource/rtl8730e/rtk_ext_flash_data.bin! =========="
+endef
+
 clean:
 	$(call DELFILE, libboard$(LIBEXT))
 	$(call CLEAN)
+ifeq ($(CONFIG_RTK_DATA_BINARY_TO_EXT_FLASH),y)
+	@rm -rf $(RESOURCE_TARGET_FOLDER)
```

Comment:
Let's use the same function for the same purpose.
```
$(call DELDIR, $(RESOURCE_TARGET_FOLDER))
```
If it's the folder, then DELDIR. 
DELFILE for file